distro/configuration/drugs/drugs-core_demo.csv

```diff
@@ -283,6 +283,7 @@ c933dc30-36f6-471f-bca6-53d8a682dce1,,Metronidazole et Nystatin Ovules Vaginales
 2b6dca2c-1499-4724-8648-42d996a57c6d,,Tilvit Sirop,8a52cbaf-7303-44a8-ba47-976c4104f9d7,94ecbbb3-71b7-407b-bac4-3279abbf1b69,N/A,
 25dc03a6-b2ca-46d5-8121-43ab025b3e9c,,Neomycine creme 5m/g,dee29eca-5758-4d7f-a98d-9a20551aaac3,0689bb6e-9371-4a0f-ad1c-63c135b1714d,5m/g,
 057862ef-9fe0-40c8-8955-3149b25c4a70,,Nifedipine Co 20mg,251a17f9-c229-4aee-9555-9bf7620b2a5a,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,20mg,
+38a33087-9fb5-451c-91e7-50c0390b54c9,,Nevirapine 50mg,80586AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,50mg
```

Comment:
Should there be a trailing comma at the end of this line (similar to the other lines) @gracepotma?